toolboxes/hdg/thermoelectric_main.cpp

```diff
@@ -102,5 +104,10 @@ int main(int argc, char *argv[])
                             }
                         );
     }
+
+    std::ofstream os ( "timers.md" );
```

Comment:
all timers are saved in the journal now, is it necessary ?

Reply:
It is not really necessary but it was easier this way. I can remove it if you want